app.js

```diff
@@ -1,140 +1,90 @@
-  },
-  calculateDailySale: function() {
-    for(var i = 0; i < time.length; i++) {
-      var customerThisHours = this.calculateNumberCustomers();
-      console.log(time[i], customerThisHours);
-      this.dailyCustomers.push(customerThisHours);
-      var salesThisHours = this.calculateHourlySales(customerThisHours);
-      console.log(salesThisHours);
-      this.dailySales.push(salesThisHours);
-    }
-    console.log(this.dailyCustomers);
-    console.log(this.dailySales);
-  }
-};
 
-var store4 = {
-  name: 'Capital Hill',
-  minCustomer: 20,
-  maxCustomer: 38,
-  avgCookies: 2.3,
-  dailySales: [], //
-  dailyCustomers: [], //
-  calculateNumberCustomers: function() {
-    return Math.random() * (this.maxCustomer - this.minCustomer) + this.minCustomer;
-  },
-  calculateHourlySales: function(customer) {
-    return customer * this.avgCookies;
-  },
-  calculateDailySale: function() {
-    for(var i = 0; i < time.length; i++) {
-      var customerThisHours = this.calculateNumberCustomers();
-      console.log(time[i], customerThisHours);
-      this.dailyCustomers.push(customerThisHours);
-      var salesThisHours = this.calculateHourlySales(customerThisHours);
-      console.log(salesThisHours);
-      this.dailySales.push(salesThisHours);
-    }
-    console.log(this.dailyCustomers);
-    console.log(this.dailySales);
-  }
+    // Calculating and attaching total cookie sales
+    // var totalSales = 0;
+    // for (var i = 0; i < this.daily.length; i++){
+    //   totalSales += this.daily[i];
+    // }
+  };
 };
 
-var store5 = {
-  name: 'Seattle Center',
-  minCustomer: 2,
-  maxCustomer: 16,
-  avgCookies: 4.6,
-  dailySales: [], //
-  dailyCustomers: [], //
-  calculateNumberCustomers: function() {
-    return Math.random() * (this.maxCustomer - this.minCustomer) + this.minCustomer;
-  },
-  calculateHourlySales: function(customer) {
-    return customer * this.avgCookies;
-  },
-  calculateDailySale: function() {
-    for(var i = 0; i < time.length; i++) {
-      var customerThisHours = this.calculateNumberCustomers();
-      console.log(time[i], customerThisHours);
-      this.dailyCustomers.push(customerThisHours);
-      var salesThisHours = this.calculateHourlySales(customerThisHours);
-      console.log(salesThisHours);
-      this.dailySales.push(salesThisHours);
-    }
-    console.log(this.dailyCustomers);
-    console.log(this.dailySales);
-  }
-};
+var h1 = document.createElement('h1');
+h1.innerText = 'All Cookie Stores';
+var main = document.getElementsByTagName('main')[0];
+main.appendChild(h1);
+
+var store1 = new Store('1st and Pike', 23, 65, 6.3);
+var store2 = new Store('SeaTac Airport', 3, 24, 1.2);
+var store3 = new Store('Seattle Center', 11, 38, 3.7);
+var store4 = new Store('Capital Hill', 20, 38, 2.3);
+var store5 = new Store('Alki', 2, 16, 4.6);
+
+store1.renderHTMLSales();
+store2.renderHTMLSales();
+store3.renderHTMLSales();
+store4.renderHTMLSales();
+store5.renderHTMLSales();
```

Comment:
Nice organization of constructor function calls and method calls. You could use better variable names for your stores though.